src/Assets/Standard Assets/Bugsnag.cs

```diff
@@ -38,6 +38,7 @@ public class NativeBugsnag {
             public static extern void ClearTab(string tabName);
         #elif UNITY_ANDROID && !UNITY_EDITOR
             public static AndroidJavaClass Bugsnag = new AndroidJavaClass("com.bugsnag.android.Bugsnag");
+            public static Regex unityExpression = new Regex ("(\\S+)\\s*\\(.*?\\)\\s*(?:(?:\\[.*\\]\\s*in\\s|\\(at\\s*\\s*)(.*):(\\d+))?", RegexOptions.IgnoreCase | RegexOptions.Multiline);
```

Comment:
When is this line called? When the app boots or when the first `Notify` is called?